.github/workflows/pull_request.yml

```diff
@@ -20,12 +20,50 @@ jobs:
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/aether/takari-local-repository/0.11.3/takari-local-repository-0.11.3.jar
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/takari-filemanager/0.8.3/takari-filemanager-0.8.3.jar
           wget -P $M2_HOME/lib/ext https://repo1.maven.org/maven2/io/takari/maven/takari-smart-builder/0.6.1/takari-smart-builder-0.6.1.jar
-      - name: free disk space
+      - name: Free disk space
         run: |
-          sudo swapoff -a
-          sudo rm -f /swapfile
-          sudo apt clean
-          docker rmi $(docker image ls -aq)
+          df -h
+          sudo apt-get remove aria2 ansible azure-cli shellcheck rpm xorriso zsync \
```

Comment:
There is one thing I am a bit worried about this space cleanup: once the `ubuntu-latest` image is updated, some of these packages will no longer be relevant. Over time, we will be able to free less and less of the disk space.

Reply:
I had the same concern, but there is only one better solution to have our own container image tailored for our usage, but that will be more effort for us.

Reply:
I agree with both of you, We will have to maintain those packages for now I'm afraid...

Reply:
btw I read a few days ago, that ubuntu latest on GA will move to 20.04 sometime in early November, so that will be first moment to catch this maintanance.